### ISSUE TITLE
Find All References doesn't work in LiveShare Guest

### ISSUE THREAD

**Issue by TanayParikh** [dotnet/roslyn#44308]
**Version Used**: 
VS Master

**Steps to Reproduce**:

1. Create a C# Console Application on VS Master
2. Paste the following into `Program.cs`

```csharp
namespace ConsoleApp
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
            Something();
        }

        static void Something()
        {

        }
    }
}
```

3. Launch a new LiveShare session, and connect to it with another VS Master instance.
4. Do Find All References on `Something();` within the VS Master LiveShare guest.

**Expected Behavior**:
See reference to the usage and declaration of `Something()`.

**Actual Behavior**:
No references shown.

Possible relation to: https://github.com/dotnet/roslyn/issues/44170

**Comment by allisonchou** [comment]
I believe what you're seeing is due to this bug: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1094786/
We're temporarily switching back to non-streaming via #44283 until the bug is fixed.

**Comment by TanayParikh** [comment]
Thanks for the clarification @allisonchou! I'll follow that work item & PR.

**Comment by allisonchou** [comment]
This should now be fixed via #45013.
